src/tests/test_utils.C

```diff
@@ -170,6 +170,9 @@ checkParameterListAgainstDefaultAndOverrides
         parameters.insert(pair.first);
     }
 
+    Teuchos::ParameterList defaultedList;
+    defaultedList.validateParametersAndSetDefaults(defaultList);
+
```

Comment:
I don't really understand why this change, but I assume there's a reason? Other than that I don't think there's anything weird in this commit.

Reply:
It makes all the parameters in `defaultedList` `[default]`. Since we use `set` in JDQZPP, and later everywhere in I-EMIC (when we start adding documentation), the values in the list that is returned from `getDefaultParameters` (`defaultList`) are not `[default]`. However, we expect the list that is being compared against to be all `[default]`, so we need these values also to be `[default]` to make them equal.

Reply:
Ah, right. Rest looks fine to me.